PWGHF/D2H/Tasks/taskXicToXiPiPi.cxx

```diff
@@ -63,6 +63,16 @@ struct HfTaskXicToXiPiPi {
 
   void init(InitContext const&)
   {
+    if (doprocessWithKFParticle == false && doprocessWithDCAFitter == false && doprocessMcWithKFParticle == false && doprocessMcWithDCAFitter == false) {
+      LOGF(fatal, "Neither doprocessWithKFParticle nor doprocessWithDCAFitter nor doprocessMcWithKFParticle nor doprocessMcWithDCAFitter enabled. Please choose one.");
+    }
+    if (doprocessWithKFParticle == true && doprocessWithDCAFitter == true) {
+      LOGF(fatal, "Cannot enable doprocessMcWithKFParticle and doprocessMcWithDCAFitter at the same time. Please choose one.");
+    }
+    if (doprocessMcWithKFParticle == true && doprocessMcWithDCAFitter == true) {
+      LOGF(fatal, "Cannot enable doprocessMcWithKFParticle and doprocessMcWithDCAFitter at the same time. Please choose one.");
+    }
```

Comment:
```suggestion
    if (!doprocessWithKFParticle && !doprocessWithDCAFitter && !doprocessMcWithKFParticle && !doprocessMcWithDCAFitter) {
      LOGF(fatal, "No process function enabled. Please enable one.");
    }
    if (doprocessWithKFParticle && doprocessWithDCAFitter) {
      LOGF(fatal, "Cannot enable doprocessMcWithKFParticle and doprocessMcWithDCAFitter at the same time. Please choose one.");
    }
    if (doprocessMcWithKFParticle && doprocessMcWithDCAFitter) {
      LOGF(fatal, "Cannot enable doprocessMcWithKFParticle and doprocessMcWithDCAFitter at the same time. Please choose one.");
    }
```
You are still accepting `doprocessWithKFParticle && doprocessMcWithDCAFitter` and `doprocessWithDCAFitter && doprocessMcWithKFParticle`.